### DISCUSSION TITLE
[Support]: Some kind of indication of a pending Frigate+ base model update?

### DISCUSSION THREAD

**Discussion by d-steve** [blakeblackshear/frigate#12099] [General Support] [answered]
### Describe the problem you are having

I'm about ready to submit for my second model request, and it occurs to me that it would be very nice to know if there's a new base model update in the next few days or so, so I can decide whether I want to get my model update in before the base model change or after. 

I think there's an argument for either case, but personally, I'd prefer to wait, as this would allow me to reduce the number of model requests I submit. I'd hate to submit a model request and a new base model to come out the next day. 

### Version

0.14.0-4e5a6eb

### Frigate config file

```yaml
Frigate+ model
```

### Relevant log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

Dahua IPC-T54IR-ZE

**Comment by tbclark3** [comment]
I have the same concern. If there were a published schedule of base model updates, it would make it easier to decide when to submit a model request.

**Comment by blakeblackshear** [comment, marked as answer]
Won't be for another month or more. Target schedule for the time being is Jan, April, July, Oct.

**Comment by pappasaj** [comment]
@blakeblackshear any word on the july release of the next base model?  Haven't seen any chatter about it yet.

**Reply by NickM-27** [reply]
see https://github.com/blakeblackshear/frigate/discussions/11106#discussioncomment-10109077
